Trap Quest.inform/Inform/Extensions/Player/Exposure Addiction.i7x

```diff
@@ -22,4 +22,26 @@ This is the knickers can't be summoned with no panties fetish rule:
 		unless wearing-target is diaper or (wearing-target is no protection and diaper lover <= 0), rule fails.
 The knickers can't be summoned with no panties fetish rule is listed in the knickers wearability rules.
 
-No Panties Addiction ends here.
+
+topless permanent fetish is a number that varies.  topless permanent fetish is 0.
+Definition: yourself is fighting against no-bra:
+	if topless permanent fetish is 1 or there is a worn topless tattoo or the humiliation of the player >= 35000:
+		let C be a random worn bra;
+		if C is bra and C is not no protection, decide yes;
```

Comment:
`no protection` is a property of crotch covering, whereas we want to use `fully exposing` here (this is one of the values of the clothing-cleavage property).  So this line 30 should read

`		if C is bra and C is not fully exposing, decide yes;`